src/styles/blocks/_header.scss

```diff
@@ -0,0 +1,39 @@
+.header {
+  box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.05);
+
+  &__container {
+    max-width: 1200px;
+    padding: 0 50px;
+    margin: 0 auto;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    gap: 20px;
+  }
+
+  &__logo {
+    position: relative;
+    width: 40px;
+    height: 40px;
+
+    img {
+      transition: all $transition;
+      position: absolute;
+      top: 0;
+      left: 0;
+      width: 100%;
+      height: 100%;
+      object-fit: cover;
+    }
+
+    &:hover img {
+      transform: scale(1.1);
+    }
+  }
+
+  &__menu {
+    @media (max-width: 767.98px) {
```

Comment:
If the styles have fractional values of sizes, then something is wrong. All sizes must be whole values.